sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -13,8 +13,9 @@
 - Moved `TimeMode`, `FeedbackQueryTimeMode`, `GetAnomaliesForDetectionConfigurationFilter`, and `GetDataFeedsFilter` from `Azure.AI.MetricsAdvisor.Models` to the `Azure.AI.MetricsAdvisor` namespace.
 - In `MetricsAdvisorClient`, renamed `GetAnomaliesForDetectionConfiguration` and `GetAnomaliesForAlert` sync and async methods to `GetAnomalies`.
 - In `MetricsAdvisorClient`, renamed `GetIncidentsForDetectionConfiguration` and `GetIncidentsForAlert` sync and async methods to `GetIncidents`.
-- In `MetricsAdvisorClient`, renamed sync and async `CreateMetricFeedback` methods to `AddMetricFeedback`.
-- In `MetricsAdvisorClient`, renamed sync and async `GetMetricFeedbacks` methods to `GetAllMetricFeedback`.
+- In `MetricsAdvisorClient`, renamed sync and async `CreateMetricFeedback` methods to `AddFeedback`.
```

Comment:
I see this was discussed in Arch board, could you remind me why? (for my own curiosity). 
Is it because methods don't need to be that specific? and since they are part of the `MetricsAdvisorClient` class user can understand that Feedback is related to `Metric`?

Reply:
Correct. According to KC, we want to make type names very specific (to avoid conflicts with other packages) and method names simpler (user already has enough context).